Utilities/bin_patch.cpp

```diff
@@ -173,3 +173,78 @@ std::size_t patch_engine::apply(const std::string& name, u8* dst) const
 
 	return found->second.size();
 }
+
+std::size_t patch_engine::apply_with_ls_check(const std::string& name, u8* dst, u32 filesz, u32 ls_addr) const
```

Comment:
I don't think another copy of a function is a good idea. Could be added to the original function.

Reply:
Will need to cleanup it later